app/models.py

```diff
@@ -1462,10 +1464,12 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
 
     def serialize(self):
         data = self.data.copy()
+        parent_brief = self.brief.serialize()
+        parent_brief_fields = ['id', 'title', 'status', 'applicationsClosedAt']
```

Comment:
@TheDoubleK
Actually wrote this up the other day 
https://docs.google.com/document/d/1mm2pEwHC18zkbJ3nec2jz94SjPeu7lv9Mo3o8f62Q5E/edit